examples/Demo/Shared/Pages/Search/Examples/SearchImmediate.razor

```diff
@@ -0,0 +1,110 @@
+﻿@using System.Timers;
```

Comment:
Could you remove this `using`, not required in the code?

Reply:
Done.